fv3config/_datastore.py

```diff
@@ -45,11 +49,15 @@ def get_orographic_forcing_directory(config):
     specified by a config dictionary.
     """
     resolution = get_resolution(config)
-    dirname = os.path.join(get_internal_cache_dir(), f"orographic_data/{resolution}")
-    if not os.path.isdir(dirname):
-        valid_options = os.listdir(
-            os.path.join(get_internal_cache_dir(), "orographic_data")
-        )
+    if 'orographic_forcing' not in config:
+        orographic_forcing_spec = 'default'
+    else:
+        orographic_forcing_spec = config['orographic_forcing']
+    parent_dirname = resolve_option(orographic_forcing_spec, OROGRAPHIC_FORCING_OPTIONS_DICT)
+    dirname = os.path.join(parent_dirname, resolution)
+
+    if get_internal_cache_dir() in dirname and not os.path.isdir(dirname):
+        valid_options = os.listdir(parent_dirname)
```

Comment:
This should behave the same whether it's local or remote - remote stores are expected to look the same as the internal cache, with resolution subdirectories. You can perform these `os` operations in a way that is also compatible with remote paths by doing `fs = filesystem.get_fs(dirname)` followed by `fs.isdir(dirname)` and `fs.listdir(parent_dirname)`. The one `os` method here that can be used on anything is `os.path.join`.